sdk/personalizer/azure-ai-personalizer/src/test/java/com/azure/ai/personalizer/PersonalizerClientTest.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.personalizer;
+
+import com.azure.ai.personalizer.models.PersonalizerRankOptions;
+import com.azure.ai.personalizer.models.PersonalizerRankResult;
+import com.azure.ai.personalizer.models.PersonalizerRankableAction;
+import com.azure.core.http.HttpClient;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static com.azure.ai.personalizer.TestUtils.DISPLAY_NAME_WITH_ARGUMENTS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PersonalizerClientTest extends PersonalizerTestBase {
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.personalizer.TestUtils#getTestParameters")
+    public void testRankThenReward(HttpClient httpClient, PersonalizerServiceVersion serviceVersion) {
```

Comment:
We should have tests for every API on the client.
Also, it would help if the test were testing a single functionality in nature.

Reply:
There are tests that single functionality already. I will remove these.